src/pages/api/feedback.ts

```diff
@@ -0,0 +1,100 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { google } from "googleapis";
+import requestIp from "request-ip";
+import { UAParser } from "ua-parser-js";
+
+export default async function feedbackFormHandler(
+  request: NextApiRequest,
+  response: NextApiResponse
+) {
+  const { method } = request;
+  if (method !== "POST") {
+    return response
+      .status(405)
+      .send({ message: "only POST requests are allowed" });
+  }
+
+  // Values to pass to spreadsheet
+  const body = request.body;
+  const type = body.category;
+  const feedbackText = body.comment;
+  // timestamp: date
+  const currentDate = new Date();
+  const currentDayOfMonth = currentDate.getDate();
+  const currentMonth = currentDate.getMonth(); // Be careful! January is 0, not 1
+  const currentYear = currentDate.getFullYear();
+  const dateString =
+    currentYear + "-" + (currentMonth + 1) + "-" + currentDayOfMonth;
+  // timestamp: time
+  const hh = currentDate.getUTCHours();
+  const mm = currentDate.getUTCMinutes();
+  const ss = currentDate.getSeconds();
+  const timeString = hh + ":" + mm + ":" + ss;
+  const timestamp = dateString + " " + timeString;
```

Comment:
The timestamp code could extracted out into a utility function such as `getTimestamp`. It seems it's also a custom timestamp for DC that is more readable for staff to read when viewing feedback? Maybe `getCustomTimestamp`?